src/dmd/objc.d

```diff
@@ -570,6 +589,9 @@ extern(C++) private final class Supported : Objc
     }
     do
     {
+        if (fd.toParent.isInterfaceDeclaration && fd.isFinal)
+            return false;
```

Comment:
No test for this case AFAICT.

Reply:
It's covered by this existing test [1]. The code is slightly refactored [2].

[1] https://github.com/dlang/dmd/blob/master/test/compilable/objc_interface_final_19654.d
[2] https://github.com/dlang/dmd/pull/11821/files#diff-e88d6d54d409ea665c7643a4a89e3fe01d8decf90042dbf8f11e905a48aabfecL1711